### PR TITLE
add chidori to urata_hrpsys_config.py and delete duplicate codes in chidori_hrpsys_config.py

### PR DESCRIPTION
**Pull request by t-karasawa**

### CODE REVIEW

File: hrpsys_ros_bridge_tutorials/src/hrpsys_ros_bridge_tutorials/urata_hrpsys_config.py [hunk 1]
```diff
@@ -271,6 +277,73 @@ def setStAbcParametersURATALEG (self):
...
+        kfp.R_angle=1000
+        self.kf_svc.setKalmanFilterParam(kfp)
+        # st setting
+        stp=self.st_svc.getParameter()
+        stp.st_algorithm=OpenHRP.StabilizerService.EEFM
+        stp.k_brot_p=[0, 0]
+        stp.k_brot_tc=[1000, 1000]
+        stp.eefm_body_attitude_control_gain=[0.5, 0.5]
+        stp.eefm_body_attitude_control_time_const=[1000, 1000]
+        stp.eefm_rot_damping_gain=20*1.6*1.1*1.5
```
Comment (snozawa) [inline hrpsys_ros_bridge_tutorials/src/hrpsys_ros_bridge_tutorials/urata_hrpsys_config.py]: Please include 
https://github.com/start-jsk/rtmros_tutorials/pull/401
commit.
Update required for `eefm_rot_damping_gain`, `eefm_pos_damping_gain`, `eefm_rot_time_const`, `eefm_pos_time_const_support`, and `eefm_leg_*_margin`.

**Comment by t-karasawa** [comment]
I'm sorry.
I corrected the mistake.

File: hrpsys_ros_bridge_tutorials/src/hrpsys_ros_bridge_tutorials/urata_hrpsys_config.py [hunk 2]
```diff
@@ -271,6 +277,94 @@ def setStAbcParametersURATALEG (self):
...
+                         OpenHRP.StabilizerService.TwoDimensionVertex(pos=[tmp_leg_front_margin, -1*tmp_leg_outside_margin]),
+                         OpenHRP.StabilizerService.TwoDimensionVertex(pos=[-1*tmp_leg_rear_margin, -1*tmp_leg_outside_margin]),
+                         OpenHRP.StabilizerService.TwoDimensionVertex(pos=[-1*tmp_leg_rear_margin, tmp_leg_inside_margin])]
+        lleg_vertices = [OpenHRP.StabilizerService.TwoDimensionVertex(pos=[tmp_leg_front_margin, tmp_leg_outside_margin]),
+                         OpenHRP.StabilizerService.TwoDimensionVertex(pos=[tmp_leg_front_margin, -1*tmp_leg_inside_margin]),
+                         OpenHRP.StabilizerService.TwoDimensionVertex(pos=[-1*tmp_leg_rear_margin, -1*tmp_leg_inside_margin]),
+                         OpenHRP.StabilizerService.TwoDimensionVertex(pos=[-1*tmp_leg_rear_margin, tmp_leg_outside_margin])]
+        stp.eefm_support_polygon_vertices_sequence = map (lambda x : OpenHRP.StabilizerService.SupportPolygonVertices(vertices=x), [rleg_vertices, lleg_vertices])
+         # stp.eefm_zmp_delay_time_const=[0.055, 0.055]
+         stp.eefm_cogvel_cutoff_freq = 4.0
```
Comment (snozawa) [inline hrpsys_ros_bridge_tutorials/src/hrpsys_ros_bridge_tutorials/urata_hrpsys_config.py]: Is this correct indent?

Comment (t-karasawa) [inline hrpsys_ros_bridge_tutorials/src/hrpsys_ros_bridge_tutorials/urata_hrpsys_config.py]: Sorry for repeated mistakes.
I corrected indent.

**Pull request merged**